dna_analyse/DNA_Analyse.py

```diff
@@ -46,9 +46,23 @@ def hole_dna_sequenz():
     return sequence
 
 
+sequence = input('Gib eine DNA-Sequenz ein: ')
 seq = hole_dna_sequenz()
 # Hier sollten jetzt die nötigen Berechnungen durchgeführt werden.
 
 # Und hier erfolgt dann die Ausgabe.
 print()
-print('Eingelesene Sequenz:')
+print('Eingelesene Sequenz:', sequence)
+print('Länge:', len(sequence))
+
+print('Base', '    ', 'Häufigkeit')
+print('G', '    ', sequence.count('G'))
+print('A', '    ', sequence.count('A'))
+print('T', '    ', sequence.count('T'))
+print('C', '    ', sequence.count('C'))
+
+GC-Gehalt= (100*(sequence.count('G')+ sequence.count('C'))/(sequence.count('G')+sequence.count('C')+ sequence.count('A')+sequence.count('T')))
```

Comment:
das Minus-Zeichen ist in Variablennamen nicht erlaubt, weil man das nicht von einer Subtraktion unterscheiden könnte